src/store/shared/shared.action.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Action creators for commonly shared data
+ * @module shared/action
+ */
+import { $injector } from '../../injection';
+import { FILE_SAVE_RESULT_CHANGED, TERMS_OF_USE_ACKNOWLEDGED_CHANGED } from './shared.reducer';
+
+const getStore = () => {
+	const { StoreService } = $injector.inject('StoreService');
+	return StoreService.getStore();
+};
+
+
+/**
+ * Set the acknowledgement of the user for the Terms of Use.
+  * @function
+ */
+export const termsOfUseAcknowledged = () => {
```

Comment:
action names should contain a verb like "setTermsOfUseAcknowledged" or "acknowledgeTermsOfUse"